### PR TITLE
feat: added Image and input Components

### PR DESCRIPTION
**Pull request by tushartiwari7**
- Images are of 3 types
  - Square 
  Square
  - Rounded
circleimg

  - Responsive
respimg

 
- Inputs are of 3 types
  - Standard Input
stdinp

  - TextBox Input
textbox

  -  Validation enabled Input element.
inputwithvalidation

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Preview seems good and.. Code is well structured.. 
Good to go 👍

**Comment by Namrata-J** [comment]
code is clean and looks good to me, you can go ahead with merging!!

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.input {
+  background: transparent;
+  color: var(--text-color);
+  border: 1px solid transparent;
```
Comment (JantuDeb) [inline components/input/input.css]: You can use `border: none;` or `border:0` as you use an outline.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes apart from that it's great.

**Review by Namrata-J — APPROVED** [review]
code is clean and looks good to me, you can go ahead with merging!!

File: components/input/input.css (hunk 1 above)
Comment (tushartiwari7) [reply components/input/input.css]: Done

**Pull request merged**